rundeckapp/src/main/groovy/com/dtolabs/rundeck/app/api/ApiVersions.groovy

```diff
@@ -40,6 +40,7 @@ class ApiVersions {
     public static final int V50 = 50
     public static final int V51 = 51
     public static final int V52 = 52
+    public static final int V53 = 53
 
```

Comment:
Since you added a new api version, the final string API_CURRENT_VERSION and API_CURRENT_VERSION_STR should also be updated to 53. I've seen that you reverted this to fix some tests failing but probably the issue with the tests was because you also need to update the current version used to build the requests from tests here: https://github.com/rundeck/rundeck/blob/4a969f7a388dcfe9a2e23252a058b2c9e4a03ec1/functional-test/src/test/groovy/org/rundeck/util/container/RdClient.groovy#L30